### PR TITLE
Add FireFly Q3 2023 report

### PR DESCRIPTION
**Pull request by nguyer**

### CODE REVIEW

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md]:
```suggestion
The Hyperledger FireFly team has been working closely with developers at OneOf over the past year as they built a new FireFly blockchain connector for the Tezos blockchain. They also demoed this new blockchain connector at a recent FireFly Community Call. This codebase is in the process of being open sourced and donated to the Hyperledger FireFly project.
```

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md:12]: Please do create an issue in the [hyperledger/toc repo](https://github.com/hyperledger/toc/issues) entitled "Move FireFly to Graduated Status" so that we can ensure that this gets added to a future meeting for discussion.

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md]: A couple of questions that will probably be asked during the request to graduate:
1. For the maintainer that transitioned to a new company, does this person's day-to-day involve work with FireFly?
2. For the maintainer from Instincttools, what is the level of maintainership?

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md]: It might be worth while renaming this image to `ff2023q3image1.png`. Including the year will ensure no conflicts next year.

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md]:
```suggestion
- FabConnect not being able to set transactionindex and eventindex
```
Not sure if this is a typo or not, but looked weird to me.

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md]:
```suggestion
- Fixed multiple named tuple result from contract
```

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md:69]: Are there any LTS releases yet?

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md:113]: Is it correct to say that we have 10 maintainers from Kaleido, 1 maintainer from Fidelity, and 1 maintainer from OneOf?

Is the 1 maintainer from Instinctools the same as that from OneOf?

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [inline project-reports/2023/2023-Q3-Hyperledger-FireFly.md]: Similar to previous comment regarding the year inclusion on the filenames.

File: project-reports/2023/images/ffq3image4.png
Comment (tkuhrt) [inline project-reports/2023/images/ffq3image4.png:1]: This shows only 2 organizations being active contributors in the past 3 months. How does this relate to the maintainer companies being 3? Is this because of the recent job move?

**Review by arsulegai — APPROVED** [review]
Please address the review comments from Tracy. All the best, looking forward to the proposal.

**Review by swcurran — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (nguyer) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md:12]: Thanks Tracy! I am working on this in parallel. Should it just be an issue here, or should it be a PR raised to https://github.com/hyperledger/hyperledger-hip ?

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (nguyer) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md]: Good suggestion. I was thinking from the directory structure that wouldn't be an issue since all the 2024 reports would be in a separate parent directory. But I can rename them if that's not the planned structure.

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (nguyer) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md:69]: Good question. We have not designated any releases as and LTS release yet, but it is something that we are actively talking about as a community. We need to define the cadence for LTS releases, how long they will be supported for etc. But it is something that we would like to do in the near future.

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (jimthematrix) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md:12]: @nguyer we clarified on the TOC call that it should be written as an issue in the TOC repo. The Cacti team has started one here: https://github.com/hyperledger/toc/issues/164 for their application

Comment (nguyer) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md:12]: Thanks for the clarifications! I will open an issue the same way. I don't have slides, but I do have a lot of Markdown I will post :)

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (nguyer) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md]: Thanks. These are good questions and I would be happy to chat about them in detail.

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (nguyer) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md:113]: > Is it correct to say that we have 10 maintainers from Kaleido, 1 maintainer from Fidelity, and 1 maintainer from OneOf?

Yes that is correct.

> Is the 1 maintainer from Instinctools the same as that from OneOf?

Yes, sorry I realize this wasn't very clear in the report. I will re-word it to make that more clear.

File: project-reports/2023/images/ffq3image4.png
Comment (nguyer) [reply project-reports/2023/images/ffq3image4.png:1]: Good question. OneOf is contributing a new repo to the project, the Tezos blockchain connector. This repo has commits dating back to March of this year. The code has cleared legal approval to be moved to the Hyperledger org, but I am still working with them to actually get the code moved over. This chart wouldn't reflect any of those contributions yet, which is why it looks the way it does today. The FireFly maintainers have reviewed the code, which was the basis for voting to have Dzianis come onboard as a maintainer. I expect the code to be moved over in the next couple of days.

**Comment by nguyer** [comment]
Thanks for the questions and feedback, everyone! I think all of it has been address, but please let me know if there are any more questions. Thanks!!

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md]: My bad. I missed that the `images` folder was already in the `2023` folder. No change needed (unless I am too late)

File: project-reports/2023/2023-Q3-Hyperledger-FireFly.md
Comment (tkuhrt) [reply project-reports/2023/2023-Q3-Hyperledger-FireFly.md:113]: If I read correctly OneOf/Instinctools is the same maintainer. Thanks for the clarification.

**Review by petermetz — APPROVED** [review]

**Comment by nguyer** [comment]
As an FYI I added a link to the newly contributed repo and some more information about the Steering Committee we are in the process of forming, as well as our Maintainer Pipeline

**Review by jimthematrix — APPROVED** [review]

**Review by lehors — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**
